_posts/2016-06-22-hello-beercraft.md

```diff
@@ -0,0 +1,52 @@
+
+Here we are at the bottling chamber. It serves also as a warehouse. 
+![Tools](/img/hello-beercraft/the_people.jpg)
+
+## Contest for label 
+
+We had our internal contest for best labels. Here is a picture of winning label. 
+![Tools](/img/hello-beercraft/intohimo.png)
+
+There were plenty of great contesters but I have to admit that I was imressed by the winner. The winner even created a 3D model that can be found [here](http://dev.makarainen.org/intohimo/)
+
+Committee choosing the winner said he was able to combine design and technical skills in a Solita like manner. Colleagues said that the look and feel was really fresh, it had artistic passion and everything fits!
+
+## Tasting notes
+
+Smell and taste has refreshing notes of passion fruit. Citra and Jarrylo hops creates bitterness in the end that is suitable for any summer beer. We were happy about how well passion fruit came out in both smell and taste. We were very pleased with the test batch thus Maku brewing will continue without making any changes to receipt. 
+
+Fellow from Solita got there before me and wrote this [blog](http://tuopillinen.blogspot.fi/2016/06/pieni-yritysvierailu-maku-brewingille.html) about the tasting in finnish. 
+
+Here is our passion pale ale in the glass. 
+![Tools](/img/hello-beercraft/glass_of_beer.jpg)
+
+## Brewing process
+
+We got introduction into Maku Brewing brewing process. They like to start their process with 20l braumeister (that was the original investment). After they have evaluated the receipt enough they change into 3000l mashing vessel. After mashing the product will be transferred into fermentation tanks. In fermentation tanks the end product will be from few weeks to months depending on the type of the beer. And finally the end product will be of course canned or bottled.
+
+I have no pictures of tanks, because well... they are tanks and not that interesting. But here are some machines with electronics! 
+
+Canning machine 
+![Tools](/img/hello-beercraft/canning_machine_2.jpg)
+
+Bottling machine
+![Tools](/img/hello-beercraft/bottling_machine.jpg)
+
+## 
+
+Our end product will be ready at the october 2016. It will be just in time for Solita's 20 years party! Now we have all set and we can just wait that our contactor finishes the project. We have receipt for great beer, design for nice outfit and known end user group that is waiting the end product to get ready. 
```

Comment:
at the october 2016 -> in August. It will be ready just in time

now we have all set -> now we are all set
receipt -> a recipe
design for nice outfit -> a nice looking design?
known end user group -> a known end user group
waiting the end product to get ready -> waiting for the end product to be ready